openedx/core/lib/block_structure/transformers.py

```diff
@@ -104,15 +104,17 @@ def is_collected_outdated(cls, block_structure):
     def transform(self, block_structure):
         """
         The given block structure is transformed by each transformer in the
-        collection, in the order that the transformers were added.
+        collection. Tranformers with filters are combined and run first in a
+        single course tree traversal, then remaining transformers are run in
+        the order that they were added.
```

Comment:
I noticed that this docstring was being outdated by our work, so I updated it in this latest commit.